ark/utils/misc_utils_test.py

```diff
@@ -91,3 +91,29 @@ def test_verify_same_elements():
         # the two lists provided do not contain the same elements
         misc_utils.verify_same_elements(one=['elem1', 'elem2', 'elem2'],
                                         two=['elem2', 'elem2', 'elem4'])
+
+
+def test_create_invalid_data_str():
+    invalid_data = ["data_" + str(i) for i in range(20)]
+
+    # Test to make sure the case of 10 invalid values creates a proper string.
+    invalid_data_str1 = misc_utils.create_invalid_data_str(invalid_data=invalid_data[:10])
+    result1 = []
+    for id in invalid_data[:10]:
+        result1.append(invalid_data_str1.find(id))
+    assert -1 not in result1
```

Comment:
You should also assert that `result1 = list(range(10))`

Reply:
`.find()` returns the first index where the search string is found, and $-1$ if it cannot find it. So its testing to make sure that up to the first 10 invalid values are in the print statement by checking if there is not a $-1$ in the `result1` list.

I could also assert that the length of the `result1` list is the same as the lengths of the `invalid_data` used.

However I'm not too sure what the assert for `result1 == list(range(10))` would mean.

Reply:
My bad, I completely missed your use of `.find`, somehow thought you were appending the raw values to `result1`. This works, you can ignore the rest of the comments I made.

Reply:
Alternatively:

```
for id in invalid_data[:10]:
    assert invalid_data_str1.find(id) != -1
```